style.css

```diff
@@ -352,3 +402,212 @@ form ::placeholder {
   font-size: 15px;
   line-height: 24px;
 }
+
+.menu ul {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .homepage {
+    background-image: url(images/Header\ bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .logo {
+    font-size: 18px;
+    line-height: 20px;
+    color: #6070ff;
+    font-weight: 700;
+    text-decoration: none;
+    padding-left: 144px;
+  }
+
+  .main_container_one {
+    width: 100%;
+    padding-top: 240px;
+    padding-left: 240px;
+    padding-right: 240px;
+  }
+
+  .main_container_one h1 {
+    width: 310px;
+  }
+
+  .menu img {
+    display: none;
+  }
+
+  .menu ul {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 30px;
+    padding-right: 154px;
+    text-decoration: none;
+  }
+
+  .menu li {
+    text-decoration: none;
+    list-style: none;
+  }
+
+  .menu a {
+    font-size: 15px;
+    font-weight: 500;
+    line-height: 20px;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .work_section {
```

Comment:
Kindly make sure the works section is responsive for different sizes. You did a great job in general, but here there is a bug needs to fixed. The work card is getting closer to the right of the screen and there is a white background showing up. Kindly check the GIF below.
<img width="1036" alt="Screenshot 2023-01-19 at 10 19 37 PM" src="https://user-images.githubusercontent.com/25887253/213540722-be5d7d42-5b79-40b4-9757-8e3c4aec6f3c.png">

![Screen Recording 2023-01-19 at 10 18 20 PM](https://user-images.githubusercontent.com/25887253/213540695-001fc98f-d994-4a12-80c1-7f1f1946cf0b.gif)